tests/characters/test_character_control.py

```diff
@@ -86,21 +86,23 @@ def test_alice_character_control_grant(alice_control_test_client, federated_bob)
 def test_alice_character_control_revoke(alice_control_test_client, federated_bob):
     bob_pubkey_enc = federated_bob.public_keys(DecryptingPower)
 
-    request_data = {
+    grant_request_data = {
         'bob_encrypting_key': bytes(bob_pubkey_enc).hex(),
         'bob_verifying_key': bytes(federated_bob.stamp).hex(),
         'label': 'test',
         'm': 2,
         'n': 3,
         'expiration': (maya.now() + datetime.timedelta(days=3)).iso8601(),
```

Comment:
Idk if important in this PR, but it was noticed that if `iso8601` is strictly followed (maya doesn't as microseconds how it does those are not in the standard), our requests fail at Ursula side (because we pass-though everything to Ursula as-is without going beyond splitting?).

Anyway, just thought: is it ok that we do that, or is it safer to re-parse at Alice API?

Reply:
Is there an open related issue for this?